src/include/duckdb/parser/result_modifier.hpp

```diff
@@ -103,6 +103,9 @@ class OrderModifier : public ResultModifier {
 	OrderModifier() : ResultModifier(ResultModifierType::ORDER_MODIFIER) {
 	}
 
+	OrderModifier(vector<OrderByNode> orders) : ResultModifier(ResultModifierType::ORDER_MODIFIER) {
```

Comment:
Shouldn't something be done with the arguments?

Reply:
ah. good catch. Removed because this constructor isn't even used